src/App.tsx

```diff
@@ -1,10 +1,58 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
+// General libs
 import React from 'react';
+import { useEffect, useState } from 'react';
+import classNames from 'classnames';
+// components
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoCreate } from './components/TodoCreate/TodoCreate';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+// api
+import { USER_ID, getTodos } from './api/todos';
+// Types
+import { Todo } from './types/Todo';
+import { StatusFilter } from './types/StatusFilter';
+import { filterTodos } from './utils/filterTodos';
+import { ErrorType } from './types/ErrorType';
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [statusFilter, setStatusFilter] = useState(StatusFilter.All);
+  const [errorType, setErrorType] = useState(ErrorType.NO_ERROR);
+
+  useEffect(() => {
+    const getTodosFromServer = async () => {
+      // I decided to try async this time. That's just for my own practice.
+      try {
+        const todos = await getTodos(); // Also, it's supposed to be a bit more efficient.
+
+        setTodoList(todos);
+      } catch {
+        // eslint-disable-next-line no-console
+        console.warn('Impossible to download the todos');
+        setErrorType(ErrorType.LOAD_TODOS);
+
+        const timer = setTimeout(() => {
+          setErrorType(ErrorType.NO_ERROR);
+          // unmounts the timer, so it won't be duplicated
+          clearTimeout(timer);
+        }, 3000);
+      }
+    };
+
+    getTodosFromServer();
```

Comment:
```suggestion
    (async () => {
      // I decided to try async this time. That's just for my own practice.
      try {
        const todos = await getTodos(); // Also, it's supposed to be a bit more efficient.

        setTodoList(todos);
      } catch {
        // eslint-disable-next-line no-console
        console.warn('Impossible to download the todos');
        setErrorType(ErrorType.LOAD_TODOS);

        const timer = setTimeout(() => {
          setErrorType(ErrorType.NO_ERROR);
          // unmounts the timer, so it won't be duplicated
          clearTimeout(timer);
        }, 3000);
      }
    })();
```

https://developer.mozilla.org/en-US/docs/Glossary/IIFE